libs/components/package.json

```
@@ -5,6 +5,6 @@
   "description": "The components library of intuiface",
   "license": "MIT",
   "scripts": {
-    "docs": "typedoc --tsconfig ./tsconfig.lib.prod.json && node ../../patch-readme.mjs ../../docs/components && npx shx cp ../../docs/components/README.md ./README.md"
+    "docs": "typedoc --tsconfig ./tsconfig.lib.prod.json && node ../../patch-readme.mjs ../../docs/components && npx shx cp ../../docs/components/README.md ./README.md && node ../../fix-links-readme.mjs ./README.md docs/components/"
```

Base path should be `../../docs/*` otherwise it's relative to `libs/*` folder. (same for command in package.json for core)
```suggestion
    "docs": "typedoc --tsconfig ./tsconfig.lib.prod.json && node ../../patch-readme.mjs ../../docs/components && npx shx cp ../../docs/components/README.md ./README.md && node ../../fix-links-readme.mjs ./README.md ../../docs/components/"
```